firmware/ventilator-controller-stm32/Core/Inc/Pufferfish/Driver/I2C/HoneywellABP.h

```diff
@@ -10,7 +10,7 @@
 #pragma once
 
 #include "Pufferfish/Driver/Testable.h"
-#include "Pufferfish/HAL/HAL.h"
+#include "Pufferfish/HAL/STM32/HAL.h"
```

Comment:
Let's only include `Pufferfish/HAL/Interfaces/I2CDevice.h` - drivers should not be including or using anything from `HAL/STM32`, so this PR is a great opportunity to fix this problem in the old code here.